metadata-editor/app/controllers/EditsController.scala

```diff
@@ -113,7 +109,9 @@ object EditsController extends Controller with ArgoHelpers {
           .setAdd(id, "labels", labels)
           .map(publish(id))
           .map(edits => labelsCollection(id, edits.labels.toSet))
-          .map {case (uri, labels) => respondCollection(labels)}
+          .map {case (uri, labels) => respondCollection(labels)} recover {
+          case _ => BadRequest
```

Comment:
(Arguably we should validate the payload before we talk to Dynamo, but OK for a quick fix for now if it's a hassle to do the validation)